### ISSUE TITLE
Fix columns in spans tab in explorer 

### ISSUE THREAD

**Issue by asldevi** [hypertrace/hypertrace#149]
Protocol, status columns are not very relevant spans and they are empty most of the time. Suggestios is to remove them from default columns in spans tab.
Instead, Span id, Trace id would be useful and helpful to keep them on by default.

<img width="1180" alt="Screenshot 2021-01-05 at 4 27 00 PM" src="https://user-images.githubusercontent.com/8813/103638856-7e13eb00-4f73-11eb-811a-84aac80c48d0.png">

**Comment by jyothishjose6190** [comment]
@kotharironak @JBAhire  Plan to pick this up in the current sprint. What are your thoughts?

**Comment by kotharironak** [comment]
This seems a similar requirement we had for time range selection (showing upto last 1 day instead of upto last 1 month). So, all these defaults need to be wrapped in some sort of meta APIs called preferences as one set may not be the same for all orgs (or consumers). Said that instead of removing existing `protocal` and `status`, we can add one more `spanId` in the default list.

**Comment by jyothishjose6190** [comment]
> This seems a similar requirement we had for time range selection (showing upto last 1 day instead of upto last 1 month). So, all these defaults need to be wrapped in some sort of meta APIs called preferences as one set may not be the same for all orgs (or consumers). Said that instead of removing existing `protocal` and `status`, we can add one more `spanId` and `traceId` in the default list. cc : @JBAhire

@kotharironak should I send a PR after adding spanId and traceId or deploy it as a separate fork?

**Comment by JBAhire** [comment]
> This seems a similar requirement we had for time range selection (showing upto last 1 day instead of upto last 1 month). So, all these defaults need to be wrapped in some sort of meta APIs called preferences as one set may not be the same for all orgs (or consumers). Said that instead of removing existing `protocal` and `status`, we can add one more `spanId` and `traceId` in the default list. cc : @JBAhire

The requirement is similar but I think this one should be more straightforward from UI perspective. If he is working with a fork, he can just add additional columns or remove existing ones and add additional columns, right?

But yeah if we want to make it configurable then we have to take same API route. :)

**Comment by kotharironak** [comment]
@jyothishjose6190 If, you are planning to deploy from a fork, change the order as well according to your requirement. But, if you are going to put it in main, would be better if we add them at the end.

**Comment by jyothishjose6190** [comment]
@JBAhire yes i saw that. I hope the item with title "Entry Span ID" is the one for the Span ID

**Comment by jaywalker21** [comment]
@jyothishjose6190 Can you update this Issue if we have worked upon this?

**Comment by jyothishjose6190** [comment]
I  have raised a PR  https://github.com/hypertrace/hypertrace-ui/pull/1178. Also released as a separate fork.
